### PR TITLE
Warn if deleting user with associated Service Account

### PR DESCRIPTION
**Pull request by jinapurapu**
Screenshot from 2022-05-20 12-44-59

### CODE REVIEW

**Review by cniackz — APPROVED** [review]
LGTM, tested locally:
image

**Comment by cniackz** [comment]
After user is deleted, service account is gone, which is good:

Screen Shot 2022-05-20 at 4 13 43 PM

**Review by reivaj05 — APPROVED** [review]

File: portal-ui/src/screens/Console/Users/DeleteUser.tsx [hunk 1]
```diff
@@ -78,14 +128,42 @@ const DeleteUser = ({
...
       onClose={onClose}
       confirmationContent={
-        <DialogContentText>
-          Are you sure you want to delete the following {selectedUsers.length}{" "}
-          user{selectedUsers.length > 1 ? "s?" : "?"}
-          <b>{renderUsers}</b>
+        <DialogContentText>  
+          
+        {hasSA ? 
+        <>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/DeleteUser.tsx:134]: Please use explicit Fragment
```suggestion
        <Fragment>
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by prakashsvmx — APPROVED** [review]

**Pull request merged**